### PR TITLE
PLATUI-855: Removed login redirect for all endpoints

### PR DESCRIPTION
**Pull request by JoPintoPaul**

### CODE REVIEW

**Comment by oscarduignan** [comment]
looks good to me, and just to say it out loud in my own words to make sure i have understood it because I'm new to the change. This removes some restrictions on certain routes, keeps all existing endpoints but consolidates the code for them, won't break any use cases and if eventually people migrate completely to the canonical endpoint then we might remove the deprecated ones

**Review by oscarduignan — DISMISSED** [review]

**Comment by JoPintoPaul** [comment]
@oscarduignan Yes exactly, basically the idea is to 1) remove the login requirement for any standalone pages, 2) retrieve enrolments for all contact requests, and 3) keep the existing endpoints until we can migrate everyone on the Platform off them.

File: app/controllers/ProblemReportsController.scala [hunk 1]
```diff
@@ -132,7 +131,7 @@ class ProblemReportsController @Inject() (
   implicit def lang(implicit request: Request[_]): Lang = request.lang
 
   //TODO default to true (or even remove the secure query string) once everyone is off play-frontend so that we use the CSRF check (needs play-partials 1.3.0 and above in every frontend)
```
[the remark this answers is not in the document]
Comment (JoPintoPaul) [reply app/controllers/ProblemReportsController.scala]: @matthewmascord Good shout. The comment was 6 years old, and I believe it was referring to an old-old library that came before `play-ui`. Either way, the endpoints `/problem_reports_secure` and `problem_reports` have exactly the same functionality now, so I've deleted the comment and removed the `externalReportProblemSecureUrl` from `AppConfig`.

**Review by oscarduignan — DISMISSED** [review]
@JoPintoPaul just to check, I missed your earlier message to reapprove and I've skimmed it again now and can see the comment removal, but are there any other differences I should look more closely at with the latest push from just now? Re-approving because I think it's just copy changes

**Comment by JoPintoPaul** [comment]
@oscarduignan Ah yes sorry, I squashed rather enthusiastically. Change to remove the comment, and to remove the use of the binding of two different URLs to the form POST for ProblemsReports partial, as the two endpoints went to the same inner method anyway (the route itself has not been deleted).

**Review by oscarduignan — APPROVED** [review]

**Review by natcarey — APPROVED** [review]

**Pull request merged**